crates/astria-core/src/primitive/v1/asset/denom.rs

```diff
@@ -691,4 +709,18 @@ mod tests {
 
         assert_eq!(None, denom.pop_leading_port_and_channel());
     }
+
+    #[test]
+    fn display_len_ok() {
+        let trace_denom = Denom::from("a/long/path/to/denom".parse::<TracePrefixed>().unwrap());
+        assert_eq!(
+            trace_denom.to_string().len(),
```

Comment:
Use the length of the input string (expressing it with the `track_caller` check I suggested above will make that neater).